morphic.js

```diff
@@ -7682,19 +7682,25 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
this could probably be done without having to introduce the optional ```submenu``` flag, don't you think?